src/styles/blocks/stars_star.css

```diff
@@ -0,0 +1,13 @@
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margins, but not both, to avoid potential margin collapse issues. If you need spacing on both sides, consider using padding or adjusting the layout to handle spacing in a more consistent manner.